strongbox-web-core/pom.xml

```diff
@@ -79,7 +73,6 @@
                         <strongbox.vault>${project.build.directory}/strongbox-vault</strongbox.vault>
                         <strongbox.storage.booter.basedir>${project.build.directory}/strongbox-vault/storages</strongbox.storage.booter.basedir>
                         <strongbox.config.xml>${project.build.directory}/strongbox/etc/conf/strongbox.xml</strongbox.config.xml>
-                        <strongbox.host>${strongbox.host}</strongbox.host>
                         <strongbox.port>${strongbox.port}</strongbox.port>
```

Comment:
Like here and in other places... :)

Reply:
@carlspring Removed strongbox.host and strongbox.port and all references.  I'm new to Maven xml files so thank you for your patience!

Reply:
Don't worry, @yupitsleen , this project is excellent for new joiners and junior developers, because we try to be a helpful and friendly bunch! Whenever you have questions, just feel free to ask and we'll try to help, or guide you.